src/schemabuilder.rs

```diff
@@ -233,21 +233,8 @@ impl SchemaBuilder {
         if indexed {
             opts = opts.set_indexed();
         }
-        let fast = match fast {
```

Comment:
These changes look like they are from the Tantivy bump?

Reply:
Rebased.